app/models/edition.rb

```diff
@@ -1,26 +1,23 @@
 class Edition < ActiveRecord::Base
   acts_as_commentable
 
-  PUBLISHERS = {
-    "Design Community" => "http://sm-11.herokuapp.com/designing-services/design-community/",
-    "Agile Community" => "http://sm-11.herokuapp.com/agile-delivery/agile-community"
-  }.freeze
-
   belongs_to :guide, touch: true
   belongs_to :user
 
   has_many :approvals
 
+  belongs_to :content_owner
+
   scope :draft, -> { where(state: 'draft') }
   scope :published, -> { where(state: 'published') }
   scope :review_requested, -> { where(state: 'review_requested') }
 
-  validates_presence_of [:state, :phase, :description, :title, :update_type, :body, :publisher_title, :publisher_href, :user]
+  validates_presence_of [:state, :phase, :description, :title, :update_type, :body, :content_owner, :user]
   validates_inclusion_of :state, in: %w(draft published review_requested approved)
   validates :change_note, presence: true, if: :major?
   validate :published_cant_change
 
-  before_validation :assign_publisher_href
+  accepts_nested_attributes_for :content_owner
```

Comment:
I don't think we need this as we do not intend to modify ContentOwner records in the UI